apps/nrm/fields/index.js

```diff
@@ -516,6 +516,24 @@ module.exports = {
       'no'
     ]
   },
+  'pv-phone-number': {
+    mixin: 'radio-group',
+    validate: 'required',
+    legend: {
+      className: 'visuallyhidden'
+    },
+    options: [{
+      value: 'yes',
+      toggle: 'pv-phone-number-yes-fieldset',
+      child: 'partials/pv-phone-number-yes'
+     }, {
+      value: 'no',
+    }]
+  },
+  'pv-phone-number-yes': {
+    mixin: 'input-text',
+    validate: 'numeric'
+  },
```

Comment:
for accessibility we have to allow text.  Users that can't hear, write please do not call me just text me at the end of their number

Reply:
No problem